core/src/main/scala/kafka/log/Log.scala

```diff
@@ -2226,12 +2231,14 @@ object Log extends Logging {
    * @param topicPartition The topic partition
    * @param logDirFailureChannel The LogDirFailureChannel to asynchronously handle log dir failure
    * @param recordVersion The record version
+   * @param logPrefix The logging prefix
    * @return The new LeaderEpochFileCache instance (if created), none otherwise
    */
   def maybeCreateLeaderEpochCache(dir: File,
                                   topicPartition: TopicPartition,
                                   logDirFailureChannel: LogDirFailureChannel,
-                                  recordVersion: RecordVersion): Option[LeaderEpochFileCache] = {
+                                  recordVersion: RecordVersion,
+                                  logPrefix: String): Option[LeaderEpochFileCache] = {
```

Comment:
Could we add the default `logPrefix` parameter value to empty string? So all the tests don't need to update.

Reply:
Introducing a default value can lead to programming error, because we could forget to pass it when it is really needed to be passed.

Reply:
Ok, good to me.